### ISSUE TITLE
TFLite Interpreter Crashes when running with TF SELECT op

### ISSUE THREAD

**Issue by Thaslim** [tensorflow/tensorflow#43934]
Hi. I have a similar issue when I try to Invoke the interpreter. 
![image](https://user-images.githubusercontent.com/3530663/95645424-a9ba2e80-0a73-11eb-8dab-30eee3b6f4eb.png)

Tflite conversion is passed without error:

converter = tf.lite.TFLiteConverter.from_saved_model('saved_model/')
converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]
converter.allow_custom_ops=True
converter.experimental_new_converter =True
tflite_model = converter.convert()
open("converted_model.tflite", "wb").write(tflite_model)
interpreter = tf.lite.Interpreter(model_path="converted_model.tflite")
interpreter.allocate_tensors()

**model definition:**

#define LSTM model with skip connection
def LSTM_net():
    N_CLASSES=4

    i = Input(shape=(79, 40), name='input')
    x = Masking()(i)
    x = LayerNormalization(name='layer_norm')(x)
    s = TimeDistributed(Dense(64, activation='tanh'),
                        name='td_dense_tanh')(x)
    x = Bidirectional(LSTM(128, return_sequences=True),
                             name='bidirectional_lstm')(s)
    x = concatenate([s, x], axis=2, name='skip_connection')
    x = Dense(64, activation='relu', name='dense_1_relu')(x)
    x = MaxPooling1D(name='max_pool_1d')(x)
    x = Dense(32, activation='relu', name='dense_2_relu')(x)
    x = Flatten(name='flatten')(x)
    x = Dropout(rate=0.5, name='dropout')(x)
    x = Dense(32, activation='relu',
                         activity_regularizer=regularizers.l2(0.001),
                         name='dense_3_relu')(x)
    o = Dense(N_CLASSES, activation='softmax', name='softmax')(x)

    model = Model(inputs=i, outputs=o, name='long_short_term_memory')

    return model

However I dont get any error when I use BatchNormalization instead of LayerNormalization. Not sure if that makes any difference.  But Model performance is not as expected with the batchnormalization. I need to use LayerNormalization for my project. Please help as soon as possible.

_Originally posted by @Thaslim in https://github.com/tensorflow/tensorflow/issues/38113#issuecomment-706484223_

**Comment by ravikyram** [comment]
@Thaslim 

Which TF version you are using?. Please, try with TF nightly version and let us know if the issue still persists.

Request you to share colab link or simple standalone code with supporting files to reproduce the issue in our environment.It helps us in localizing the issue faster.Thanks!

**Comment by Thaslim** [comment]
Hi I used the latest tf nighty  version 2.4.0-dev20201012. 
Code to reproduce the issue.[https://colab.research.google.com/gist/jvishnuvardhan/bbde9d5ac7e8e66384ef2c5a9a19e31b/tfliteconversion.ipynb](url)

**Comment by ravikyram** [comment]
I have tried in colab with TF nightly version(`2.4.0-dev20201012`) and was able to reproduce the issue.Please, find the gist [here](https://colab.research.google.com/gist/ravikyram/652a582e77c7c8961864e8010a5c9b7c/untitled445.ipynb).Thanks!

**Comment by Thaslim** [comment]
Any updates on this issue?

**Comment by renjie-liu** [comment]
We recently support lower FusedBatchNormV3 in mlir, can you install tf-nightly and try again with the model conversion?

Thanks

**Comment by karimnosseir** [comment]
@Thaslim You're using SELECT_TF_OPS that means for operations that TFLite can't support natively, we will fallback to TF op.
In this case, you will need to use the Interpreter which supports SELECT mode.
See https://www.tensorflow.org/lite/guide/ops_select

The reason we have separate versions of Interpreter (Supports SELECT and one with only Native TFLite operations) is because SELECT mode comes with binary size impact, so users who are ok with the extra size and want to use SELECT will need to use the Interpreter version which supports SELECT, the link above have more details and how to use it.

**Comment by Thaslim** [comment]
Notebook crashes when I try with the following assignment as mentioned in the ink
 converter.target_spec.supported_ops = [
  tf.lite.OpsSet.TFLITE_BUILTINS, # enable TensorFlow Lite ops.
  tf.lite.OpsSet.SELECT_TF_OPS # enable TensorFlow ops.
]

**Comment by karimnosseir** [comment]
Terry can you please check the crash with TF Select during invoke

Thanks

**Comment by sachinprasadhs** [comment]
Was able to reproduce your issue in Tf Nightly 2.6.0-dev20210601, please find the gist [here](https://colab.research.google.com/gist/sachinprasadhs/d12488de0c052d70675ac1c19f254e90/43934.ipynb). Thanks!

**Comment by karimnosseir** [comment]
@sachinprasadhs in your example, you are creating a tensor with wrong shape than what the model expects. it should be
[1, 79, 20]

```
input_data = np.array( np.random.random((1, 79,20)), dtype=np.float32)
```

Changing it works

**Comment by jvishnuvardhan** [comment]
Agree with @karimnosseir that this is resolved. [Here](https://colab.research.google.com/gist/jvishnuvardhan/f39e1aa20e408e946e170aff2f8c9f88/43934.ipynb) is a gist with recent `tf-nightly`.

I am closing the issue as this was resolved. Please feel free to reopen if the issue persists again. Thanks!
